app/src/main/res/layout/settings_activity.xml

```diff
@@ -3,11 +3,10 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
-    <FrameLayout
-        android:id="@+id/settings"
+    <androidx.fragment.app.FragmentContainerView
+        android:id="@+id/settings_container"
         android:layout_width="match_parent"
-        android:layout_height="0dp"
-        android:layout_weight="1" />
```

Comment:
`match_parent` is not equivalent to the removed lines and it would hide the adContainer below